src/timesync/timesyncd-server.c

```diff
@@ -92,6 +92,14 @@ int server_name_new(
                 break;
         case SERVER_LINK:
                 LIST_APPEND(names, m->link_servers, n);
+                // If we add a new link server, emit changed event
+                if (sd_bus_is_ready(m->bus) > 0)
+                    (void) sd_bus_emit_properties_changed(
+                        m->bus,
+                        "/org/freedesktop/timesync1",
+                        "org.freedesktop.timesync1.Manager",
+                        "LinkNTPServers",
+                        NULL);
```

Comment:
indentation is off. please follow coding style

Reply:
also, if you add this, add this please for all four properties, i.e. SystemNTPServers, FallbackNTPServers, RuntimeNTPServers and LinkNTPServers

Reply:
instead of submitting the propertieschanged signal here directly, please allocate a "defer" event source for the event loop instead, that fires the events, and simple enable it here, so that you can coalesce events here, so that you only send out one event for all servers changed.

Reply:
> 
Sorry for late reply.
I'm not familiar with the "defer" event, have change the code refer to other "defer" events, but not catch the property-change signal and I don't know why
Please help check the problem when you feel free, many thanks!